spoilers/frontend-step-05.md

```diff
@@ -1,11 +1,9 @@
 ```jsx
 if (
-    selectedAlternateShippingInstruction !== 'other' ||
-    otherShippingValue !== ''
+  selectedAlternateShippingInstruction !== 'other' ||
+  otherShippingValue !== ''
 ) {
-    if ( validationError ) {
```

Comment:
The indent changes are good, but not sure about removing the `if` - clearing the validation error every time is slightly more expensive than doing this check first.

Is there a specific reason you removed the check?

Reply:
Because this `if` is a subject of [frontend-step-07-extra-credit].